app/controllers/internal/invitations_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Internal
+  class InvitationsController < Internal::ApplicationController
+    layout "internal"
+
+    def index
+      @invitations = User.where(registered: false).page(params[:page]).per(50)
+    end
+
+    def new; end
+
+    def create
+      email = params[:user][:email]
+      name = params[:user][:name]
+      username = name.downcase.tr(" ", "_") + rand(1000).to_s
```

Comment:
**nitpick (non-blocking):** During one of the last cooldowns I changed most code like this, would be great if we can keep it consistent:


```suggestion
      username = "#{name.downcase.tr(" ", "_")}#{rand(1000)}"
```